zoo/example.txt

```diff
@@ -0,0 +1,24 @@
+Basic usage:
+
+Set up of a simple zoo with two cages for testing.
+
+from zoo.zoo import Zoo, Cage, BaseSpecies, Lion, Hyena, Wildebeest, Gazelle
+zoo = Zoo('Test Zoo')
+zoo.__repr__  # Displays the name of the zoo alongside the current number of cages and animals
+one_cage = zoo.add_cage('One Cage')  # Cages can be created directly from a Zoo object passing only their name.
+another_cage = zoo.add_cage('Another Cage')
+
+one_cage.__repr__  # Displays the cage name alongside the animals that are currently in it.
+one_cage.add_animal(Lion('Lilly'))  # Animals can be created directly from a cage or independently.
```

Comment:
Nice.  Again, another way could be:

   lion = Lion('Lilly')
   lion.add_to_cage(one_cage)

Have a think about strengths and weaknesses of either.